### PR TITLE
High scores

### PR DESCRIPTION
**Pull request by karinamzalez**
- add score board prototype 
- render scores in descending order beneath the canvas 
- currently not using addHighScoreToPage 
  - will be used when the end-game sequence is added 
- ALL 58 TESTS PASSING

### CODE REVIEW

File: lib/score-board.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  descendingScores.forEach( function(score){
+    var scoreItem = document.createElement('li');
+    scoreItem.innerText = score;
+    list.appendChild(scoreItem);
+  });
+};
+
+ScoreBoard.prototype.addHighScoreToPage = function(score){
+  var scoreItem = document.createElement('li');
+  scoreItem.innerText = score;
```
Comment (chompasina) [inline lib/score-board.js:19]: Nice work!

File: test/score-board-test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// const chai = require('chai');
+// const assert = chai.assert;
+// chai.use(require('chai-dom'));
+// const ScoreBoard = require('../lib/score-board');
+//
+// describe('ScoreBoard', function(){
+//   context('with default attributes', function(){
+//     var scoreBoard = new ScoreBoard();
+//     it('should set html element as list', function(){
+//       var ulElement = document.getElementById('high-scores');
```
Comment (chompasina) [inline test/score-board-test.js:10]: So this is a WIP I'm guessing? Does it find the ul element at all?

**Comment by chompasina** [comment]
Approved! Local storage: 💯 !

File: lib/score-board.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+function ScoreBoard(list){
+  this.list = list;
+  this.scores = [];
+}
+
+ScoreBoard.prototype.loadStoredScores = function(){
+  this.scores = localStorage.getItem('scores').split(",");
+  var list = this.list;
+  var descendingScores = this.scores.reverse();
```
Comment (deborahleehamel) [inline lib/score-board.js:9]: Did we want to limit the scores that display to top 5?

**Comment by deborahleehamel** [comment]
Excited we got this local storage working to save scores. Let's revisit if it will be better to have player enter their names at the start of a game or at the end of the game when/if they qualify for a high scorer.

**Pull request merged**